library/ssl_tls13_server.c

```diff
@@ -138,58 +215,47 @@ static int ssl_tls13_offered_psks_check_identity_match_ticket(
     }
```

Comment:
(nit; pre-existing) the debug message "buffer too small" seems wrong for `calloc()` failure

Reply:
I've just removed it as we usually do not log memory allocation failures.